### PR TITLE
auth revoke

### PR DESCRIPTION
**Pull request by aryanbhosale**

### CODE REVIEW

**Comment by prasadtalasila** [comment]
@aryanbhosale The signout does not happen. The application behavior is same as before.

File: client/src/util/auth/Authentication.ts [hunk 1]
```diff
@@ -29,7 +32,10 @@ export async function signOut() {
     localStorage.clear();
     sessionStorage.clear();
 
+    auth.stopSilentRenew();
```
[the remark this answers is not in the document]
Comment (aryanbhosale) [reply client/src/util/auth/Authentication.ts:35]: okay sir will do

[the remark this answers is not in the document]
Comment (aryanbhosale) [reply client/src/util/auth/Authentication.ts:35]: > Please only use the functions that are needed and remove the rest.

the stopSilent function ensures that it doesnt try to renew the token once it's logged out

File: client/src/util/auth/Authentication.ts [hunk 2]
```diff
@@ -7,6 +7,9 @@ import { getLogoutRedirectURI } from '../envUtil';
 export interface CustomAuthContext {
   signoutRedirect: () => Promise<void>;
   removeUser: () => Promise<void>;
```
[the remark this answers is not in the document]
Comment (aryanbhosale) [reply client/src/util/auth/Authentication.ts:9]: these are just placeholders that i created as i was reading through the docs of oidc, later realising that theyre not used anywhere

**Pull request closed without merging**